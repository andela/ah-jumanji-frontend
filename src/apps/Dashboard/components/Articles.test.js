import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import Articles from'./Articles';


Enzyme.configure({ adapter: new Adapter() });

describe('Articles', () => {
  it('should render correctly', () => {
    const component = shallow(<Articles actions={{type: "Call-to-action"}} articles={{article:"this is article"}} />);

    expect(component).toMatchSnapshot();
  });
});