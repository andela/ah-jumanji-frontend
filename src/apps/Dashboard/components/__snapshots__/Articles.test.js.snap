// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Articles should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
